doc/5.reference/pd-messages.pd

```diff
@@ -555,51 +553,45 @@
 #X connect 17 1 8 0;
 #X connect 18 0 10 0;
 #X coords 0 -1 1 1 80 170 2 50 50;
-#X restore 519 373 pd label;
-#X obj 487 579 t b s;
-#X text 366 283 ('chocolate' is for both selection and GOP).;
-#X text 373 187 The selection color affects the outline and text of boxes. The GOP rectangle represents the viewing area of a GOP subwindow (like the ones below \, open them to check it out)., f 43;
-#X text 91 611 set colors -->;
-#X text 411 679 You can also set colors via startup with the "-send" flag \, for instance: -send "pd colors black white blue", f 36;
-#X msg 86 249 colors grey80 purple magenta red;
-#X msg 67 191 colors grey black red blue;
-#X msg 24 114 colors #000000 #ffffff #0000ff #ff0000;
-#X text 234 137 (default);
-#X msg 56 169 colors #808080 #000000 #FF0000 #0000FF;
-#X msg 79 225 colors #cccccc #800080 #ff00ff #ff0000;
-#X msg 200 685 \; pd colors \$1 \$2 \$3 \$4;
-#X msg 108 309 colors #8B7765 #FFE4E1 #D2691E;
-#X text 39 450 Note that letters can be either lower or upper case., f 17;
-#X msg 176 436 list #000000 #FfFfFf #0000FF #ff0000, f 7;
-#X text 37 7 You can set colors for foregroung \, background \, selection and (optionally) GOP rectangle. The default values for these are "black" \, "white" \, "blue" and "red". If not set \, the 4th color (GOP rectangle) takes the value of the 3rd color (selection). You should use RGB values in hexadecimal starting with "#" as a best practice \, but while Pd still uses tcl/tk \, color names are also supported (see https://www.tcl-lang.org/man/tcl/TkCmd/colors.htm for a full list - just don't use the versions with space)., f 91;
+#X restore 517 348 pd label;
+#X obj 485 554 t b s;
+#X text 371 191 The selection color affects the outline and text of boxes. The GOP rectangle represents the viewing area of a GOP subwindow (like the ones below \, open them to check it out)., f 43;
+#X text 89 586 set colors -->;
+#X text 409 654 You can also set colors via startup with the "-send" flag \, for instance: -send "pd colors black white blue", f 36;
```

Comment:
this now (as in current `master` aka ae6ad11a1330f8cc90f3bae35a4672cfcd4fa156) throws errors.

also https://github.com/pure-data/pure-data/blob/ae6ad11a1330f8cc90f3bae35a4672cfcd4fa156/doc/1.manual/resources/chapter5.htm#L54 needs to be updated to no longer mention color names.

Reply:
These have been already fixed by https://github.com/pure-data/pure-data/pull/2660/commits/89ff15479bf940a8b185c1609b5c3b2c406b349d and https://github.com/pure-data/pure-data/pull/2660/commits/174748b37e0190d55a5346704336b887f1c28f07 ?

Reply:
no.

the tip of #2660 (as of now this is 174748b37e0190d55a5346704336b887f1c28f07) still says
> -send "pd colors black white blue"